dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,339 @@
 <template>
```

Comment:
Btw I think the files should be named as:
`CreateEuTaxonomyForFinancials.vue` and
`CreateEuTaxonomyForNonFiancials.vue` 
as we write "Eu" in our backend models and classes too instead of "EU"

Reply:
ok